packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -50,9 +50,9 @@ class SlackBridge {
 			this.registerForSlackEvents();
 			RocketChat.settings.get('SlackBridge_Out_Enabled', (key, value) => {
 				if (value) {
```

Comment:
Too much duplicate code and easy to fat finger the event names.  Isolated these to their own methods.